### DISCUSSION TITLE
Would pre-registration be of interest for the community?

### DISCUSSION THREAD

**Discussion by kjappelbaum** [kjappelbaum/mofdscribe#296] [Ideas]
A practice that is quite common for (clinical) studies is that one registers the study (with the hypothesis one aims to investigate) before conducting the study. One reason for doing that is to [minimize the potential for p-hacking](https://lakens.github.io/statistical_inferences/prereg.html). One issue people cite is that it tends to be hard to publish negative results. 

There are now several efforts, e.g. 

* https://preregister.science/
* https://aspredicted.org/
* https://osf.io/zab38/wiki/home/

that attempt to introduce pre-registration for ML/general sciences. 

Would it be of interest for the digital reticular chemistry community? What would we need to build to make this usable?

**Comment by sgbaird** [comment]
Maybe in a similar vein is [The Journal of Brief Ideas](https://beta.briefideas.org/). I tried this out, too https://beta.briefideas.org/ideas/12372397dbaf594ca372f17ebbb8c2a3. I'm a fan of ideas getting out there as early as possible in the stage of development, but I also know that I still have some hesitancy sometimes in sharing some ideas publicly.
